packages/cli/src/commands/link/unlink.js

```diff
@@ -137,7 +136,9 @@ function unlink(args: Array<string>, ctx: ConfigT) {
     })
     .catch(err => {
       logger.error(
-        `It seems something went wrong while unlinking. Error:\n${err.message}`,
+        `It seems something went wrong while unlinking. CLIError:\n${
+          err.message
+        }`,
```

Comment:
How about actually throwing `CLIError` here?

Reply:
yeah, could be.